sources/util.ts

```diff
@@ -56,6 +56,16 @@ export function isHTMLLegendElement(
 	return isElement(node) && node.tagName === "LEGEND";
 }
 
+export function isHTMLSvgElement(node: Node | null): node is SVGElement {
```

Comment:
That's not necessarily true. `SVGElement` is the interface for elements inside `<svg>`. In `<svg><title /></svg>` `svg` and `title` are `SVGElement`s.

What you probably mean is 
```suggestion
export function isSVGSVGElement(node: Node | null): node is SVGSVGElement {
```